web/themes/custom/nica_theme/css/style.css

```diff
@@ -7318,3 +7318,22 @@ p:last-child,
 .control-group .help-inline:first-child {
   margin-top: 0;
 }
+
+.views-rest-feed {
```

Comment:
@edysmp are we using LESS for this site? Should these go into an appropriate less file to be generated?